### DISCUSSION TITLE
Licence

### DISCUSSION THREAD

**Discussion by ericklb84** [vtex/styleguide#1421] [General]
# Licence declaration

The vtex styleguide isn't open source? If is community, what the licence? MIT, Apache, GPL?

**Comment by estacioneto** [comment]
Thanks for opening this discussion, @ericklb84!

@kevinch, can you update us on this matter?

**Reply by kevinch** [reply]
Working on it. Should be settled soon.

**Reply by kevinch** [reply]
PR open: https://github.com/vtex/styleguide/pull/1422

**Reply by kevinch** [reply]
License added.
